### PR TITLE
Add alley road class

### PR DESCRIPTION
**Pull request by brad-richardson**
# Description

Adds alley as an additional road class, which is the last remaining major service road type (1.7M/7% of service roads).

# Reference

1. [Internal issue](https://github.com/OvertureMaps/tf-transportation/issues/84)

# Testing

Ran tests with sample alley from segment data, included as example.

# Checklist

1. [X] Add relevant examples.
2. [ ] Add relevant counterexamples.
3. [ ] Update in-schema documentation using plain English written in complete sentences, if an update is required.
4. [ ] Update Docusaurus documentation, if an update is required.
5. [ ] Review change with Overture technical writer to ensure any advanced documentation needs will be taken care of, unless the change is trivial and would not affect the documentation.

# Documentation Website

[Docs preview for this PR.](https://dfhx9f55j8eg5.cloudfront.net/pr/134)

### CODE REVIEW

File: examples/transportation/segment/road/road-alley.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  subtype: road
+  road:
+    class: alley
+    restrictions:
+      access:
+        - allowed:
+            when:
+              using: [at_destination]
+      speed_limits:
+        - max_speed: [15, "mph"]
```
Comment (vcschapp) [inline examples/transportation/segment/road/road-alley.yaml:22]: Nit: I would be tempted to drop this to simplify the example.

**Review by vcschapp — APPROVED** [review]
```
                  __..-======-------..__
              . '    ______    ___________`.
            .' .--. '.-----.`. `.-----.-----`.
           / .'   | ||      `.` \\     \     \\            _
         .' /     | ||        \\ \\_____\_____\\__________[_]
        /   `-----' |'---------`\  .'                       \
       /============|============\'-------------------.._____|
    .-`---.         |-==.        |'.__________________  =====|-._
  .'        `.      |            |      .--------.    _` ====|  _ .
 /     __     \     |            |   .'           `. [_] `.==| [_] \
[   .`    `.  |     |            | .'     .---.     \      \=|     |
|  | / .-. '  |_____\___________/_/     .'---. `.    |     | |     |
 `-'| | O |'..`------------------'.....'/ .-. \ |    |       ___.--'
LGB  \ `-' / /   `._.'                 | | O | |'___...----''___.--'
      `._.'.'                           \ `-' / [___...----''_.'
                                         `._.'.'
```

File: examples/transportation/segment/road/road-alley.yaml (hunk 1 above)
Comment (brad-richardson) [reply examples/transportation/segment/road/road-alley.yaml:22]: I just checked and we have speed limits as low as 0 (obviously bad data) but many 1 km/h with 2.6K having a numeric value <5. I propose we drop it to 1 to just restrict truly invalid numbers.

Comment (vcschapp) [reply examples/transportation/segment/road/road-alley.yaml]: I agree. I'm OK with filing an issue or just dropping it in this PR, whatever makes most sense.

**Review by vcschapp — DISMISSED** [review]

File: examples/transportation/segment/road/road-alley.yaml (hunk 1 above)
Comment (brad-richardson) [reply examples/transportation/segment/road/road-alley.yaml]: I created a separate issue, we have a variety of speed limit issues that deserve a little more thought and it limits this PR to the original intent.

**Review by sanjibdutta1 — DISMISSED** [review]

File: examples/transportation/segment/road/road-alley.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  update_time: "2023-02-23T00:03:59-08:00"
+  version: 4
+  subtype: road
+  road:
+    class: alley
+    restrictions:
+      access:
+        - allowed:
+            when:
+              using: [at_destination]
```
[the remark this answers is not in the document]
Comment (brad-richardson) [reply examples/transportation/segment/road/road-alley.yaml]: Good catch, I've updated the example

**Review by bastiaanvanassche-tomtom — APPROVED** [review]

**Review by vcschapp — APPROVED** [review]

**Pull request merged**